maybe/maybe_unix.go

```diff
@@ -0,0 +1,19 @@
+package maybe
+
+import (
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/google/renameio"
+)
+
+// WriteFile mirrors ioutil.WriteFile. On Linux it replaces an existing file
+// with the same name atomically. On Windows the file is not written atomically
+// and most permission bits are ignored.
+func WriteFile(filename string, data []byte, perm os.FileMode) error {
```

Comment:
Could you add a paragraph stating when it is appropriate to use this function? Something like:

Prefer using renameio.WriteFile instead so that you get an error if atomic replacement is not possible on the runtime platform. maybe.WriteFile is meant as a convenience wrapper if you are okay with atomic replacement not being supported by the runtime platform.

Reply:
Done.